zsl/interface/run.py

```diff
@@ -8,91 +8,180 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from builtins import object
+import sys
 
+import click
+click.disable_unicode_literals_warning = True
 
-# Initialize the path
-import sys
-import os
+from zsl import Zsl, __version__
 
-# TODO: Consider removing automatic path initialization!
-sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
 
-# Initialize
-from zsl.interface.importer import initialize_web_application
+def get_version(ctx, _, value):
+    """Click callback for option to show current ZSL version."""
 
-initialize_web_application()
+    if not value or ctx.resilient_parsing:
+        return
+    message = 'Zsl %(version)s\nPython %(python_version)s'
+    click.echo(message % {
+        'version': __version__,
+        'python_version': sys.version,
+    }, color=ctx.color)
+    ctx.exit()
 
-import json
-from zsl.router import task_router
-from zsl.task.job_context import JobContext
-from zsl.application import service_application
 
-conf = service_application.config
+@click.group(help="ZSL cli toolkit")
+@click.option('--version', help="Show ZSL version", expose_value=False,
+              is_flag=True, is_eager=True, callback=get_version)
+def cli():
+    pass
 
 
-class Job(object):
-    def __init__(self, data):
-        self.data = {'data': data}
+@cli.group(help="run something useful")
+def run():
+    pass
```

Comment:
I see, cool :). But maybe something like a warning should be outputted - please choose a worker type or give a command - so that the user knows that something else should be done.